### ISSUE TITLE
64 bit bestspeech addon for NVDA

### ISSUE THREAD

**Issue by M3198** [samtupy/b32tts_wrapper#8]
Hello,

We're on NVDA 2026.1 and we still don't have a bestspeech addon compatible. I tried Chris Duffley's fork, every time I try to load it, I get synthesizer error cannot load speech synthesizer. Please release an update to make it work with NVDA 2026.
Best regards,
Muhammad.

**Comment by joshknnd1982** [comment]
Hopefully Sam will put this addon on his site. bestSpeech 64-bit. file attached. 

[BeSTspeech-64-bit-NVDA.zip](https://github.com/user-attachments/files/29161332/BeSTspeech-64-bit-NVDA.zip)

**Comment by hmc6505** [comment]
This linked addon has flaky behavior when Rate Boost is checked.

1. Install the addon in 64-bit NVDA.
2. Check Rate Boost, set rate slider to 70% (happens at slower rates but is more prone the faster the rate).
3. Move around, spell stuff, interrupt speech, do normal computer things.

Expected: Normal behavior.

Actual: Speech gets cut off, sometimes the spoken text is one string behind what NVDA sends.

Reason: B32 helper application dies before the text is done speaking. The addon driver tries to restart the helper but it's janky and still doesn't address the underlying problem.
